calico_containers/pycalico/datastore_datatypes.py

```diff
@@ -391,6 +391,8 @@ class Rule(dict):
                     "dst_net",
                     "icmp_type",
                     "icmp_code",
+                    "icmpv6_type",
```

Comment:
I don't think this is correct.

The source for calico suggests that for ICMP and ICMPv6 the code and type field are just called icmp_code and icmp_type.